### ISSUE TITLE
SeleniumActions is somewhat misleading

### ISSUE THREAD

**Issue by furti** [porscheinformatik/selenium-components#1]
The name of at.porscheinformatik.seleniumcomponents.SeleniumActions is misleading in my opinion. The selenium webdriver itself comes with Actions (MouseAction, KeyboardAction,...).

But this is a simple utility class with helper methods for components and has nothing to do with seleniums actions.

**Comment by thred** [comment]
Yes, that's correct, but SeleniumUtils is already taken. I would be thankful for a better name, that's not *Utils. The class itself is quite complete now and it should go hand in hand with the SeleniumAsserts class.
SeleniumUtils isn't the best name either. The utilities are not directly linked to Selenium. Maybe we rename SeleniumUtils to something else and use the name for the actions class.
